client/src/stylesheets/components/_CampaignPage.scss

```diff
@@ -193,22 +173,11 @@ row {
     margin-bottom: 10px;
   }
 
-  .tools {
-    font-size: 14px;
-    text-align: center;
-    margin-left: 8.3333333333%;
-    margin-bottom: 3em;
-    & i {
-      color: $maingreen;
-      font-size: 8px;
-    }
-  }
-
   .status {
     margin-top: 1.5em;
   }
 
-  .status-bar {
+  .status-bar:not(:first-child) {
```

Comment:
what is this :not(:first-child) accomplishing?  
Perhaps include a quick comment?

Reply:
Looks like obsolete code now. It was preventing duplicate margin-top spacing.